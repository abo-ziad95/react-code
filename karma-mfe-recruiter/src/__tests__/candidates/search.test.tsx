import React from 'react';
import renderer from 'react-test-renderer';
import Karma from "@hatech/karma-core";
import Search from "../../views/candidates/search";

it("Search component renders correctly", () => {
    const search = renderer
        .create(<Karma><Search/></Karma>)
        .toJSON();
    expect(search).toMatchSnapshot();
});