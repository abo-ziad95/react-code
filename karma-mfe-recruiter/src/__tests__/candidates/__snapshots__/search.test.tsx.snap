// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search component renders correctly 1`] = `
Array [
  <div
    className="makeStyles-searchWrap-1"
  >
    <form
      className="makeStyles-search-2"
      onSubmit={[Function]}
    >
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-12 MuiGrid-grid-md-12 MuiGrid-grid-lg-12 MuiGrid-grid-xl-12"
      >
        <div
          className="MuiFormControl-root MuiTextField-root MuiFormControl-marginNormal MuiFormControl-fullWidth"
        >
          <label
            className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
            data-shrink={false}
            htmlFor="search"
          >
            Search
          </label>
          <div
            className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-fullWidth MuiInput-fullWidth MuiInputBase-formControl MuiInput-formControl"
            onClick={[Function]}
          >
            <input
              aria-invalid={false}
              autoComplete="off"
              className="MuiInputBase-input MuiInput-input"
              disabled={false}
              id="search"
              name="search"
              onAnimationStart={[Function]}
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              required={false}
              type="text"
              value=""
            />
          </div>
        </div>
      </div>
      <button
        className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
        disabled={false}
        onBlur={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
        type="submit"
      >
        <span
          className="MuiButton-label"
        >
          Search
        </span>
      </button>
    </form>
  </div>,
  <div
    style={
      Object {
        "height": "calc(100vh - 66px)",
      }
    }
  >
    <div
      onClick={[Function]}
      onMouseDownCapture={[Function]}
      onMouseMove={[Function]}
      style={
        Object {
          "height": "100%",
          "margin": 0,
          "padding": 0,
          "position": "relative",
          "width": "100%",
        }
      }
    >
      <div
        style={
          Object {
            "height": "100%",
            "left": 0,
            "margin": 0,
            "padding": 0,
            "position": "absolute",
            "top": 0,
            "width": "100%",
          }
        }
      />
      <div
        style={
          Object {
            "height": "50%",
            "left": "50%",
            "margin": 0,
            "padding": 0,
            "position": "absolute",
            "top": "50%",
            "width": "50%",
          }
        }
      >
        <div
          style={
            Object {
              "height": "100%",
              "left": 0,
              "margin": 0,
              "padding": 0,
              "position": "absolute",
              "top": 0,
              "width": "100%",
            }
          }
        />
      </div>
    </div>
  </div>,
]
`;
